validator/src/commands/run/args/rpc_bootstrap_config.rs

```diff
@@ -0,0 +1,59 @@
+use {
+    crate::commands::{FromClapArgMatches, Result},
+    clap::{value_t, ArgMatches},
+};
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct RpcBootstrapConfig {
+    pub no_genesis_fetch: bool,
+    pub no_snapshot_fetch: bool,
+    pub check_vote_account: Option<String>,
+    pub only_known_rpc: bool,
+    pub max_genesis_archive_unpacked_size: u64,
+    pub no_incremental_snapshots: bool,
+}
+
+impl Default for RpcBootstrapConfig {
+    fn default() -> Self {
+        Self {
+            no_genesis_fetch: false,
+            no_snapshot_fetch: false,
+            check_vote_account: None,
+            only_known_rpc: false,
+            max_genesis_archive_unpacked_size: 10485760,
+            no_incremental_snapshots: false,
+        }
+    }
+}
```

Comment:
`Default` should be for tests only

Reply:
my plan is using all `Default` to replace `DefaultArgs` which lives in the cli.rs. however, I can propose it later. [23b13ed](https://github.com/anza-xyz/agave/pull/6667/commits/23b13edc9e65c4ea8915ea083d1c32fb5e6c0209)